internal/catalog/build.go

```diff
@@ -304,7 +304,9 @@ func Update(c *pg.Catalog, stmt nodes.Node) error {
 
 	case nodes.CreateSchemaStmt:
 		name := *n.Schemaname
-		if _, exists := c.Schemas[name]; exists {
+		// It's possible for the sqlc-internal scheme to be created by the user
+		// explicitly to allow static analysis of code with named params.
+		if _, exists := c.Schemas[name]; exists && name != "sqlc" {
```

Comment:
I think is better solved a different way. If the user needs to create the `sqlc` schema, they should do it in such a way that it does not conflict. Specifically:

```sql
CREATE SCHEMA IF NOT EXISTS sqlc; 
```

Reply:
Thanks!

I knew I should have read documentation first.